setup.py

```diff
@@ -34,24 +36,45 @@
 ]
 
 INSTALL_REQUIRES = [
-    'numpy >= 1.12',
-    'pandas >= 0.23.0, <1.0.0',
+    'matplotlib >= 3.0.0',
+    'numpy >= 1.15',
+    # exclude pandas==1.0.0 & 1.0.1 for GH142, and 0.24.0 for GH114
+    'pandas >= 0.23.0,!=0.24.0,!=1.0.0,!=1.0.1',
     'statsmodels >= 0.8.0',
     'scipy >= 0.19.1',
     'h5py >= 2.7.1',
-    'pvlib >= 0.5.0, <0.6.0',
+    'pvlib >= 0.7.0, <0.8.0',
```

Comment:
We should consider allowing pvlib 0.8.x; I think the only issue is the deprecated `LocalizedPVSystem` class, which we've already deprecated for the rdtools 2.x series.

Reply:
I think that makes sense